webapps/camunda-commons-ui/lib/widgets/selection-type/cam-widget-selection-type.template.html

```diff
@@ -0,0 +1,50 @@
+<div class="form-inline">
+  <div>
+    <label for="instance"></label>
+    <input type="radio"
+           id="instance"
+           name="selection_type"
+           value="INSTANCE"
+           ng-model="selectionType"
+           ng-change="updateSelectionType(selectionType)" />
```

Comment:
```suggestion
```
We could get rid of this event listener (and also the one for the "query" radio button) by listening on the change of the model in the controller like this:
```javascript
const updateSelectionType = scope.updateSelectionType = function(selectionType) {
  // ...
};

scope.$watch('selectionType', updateSelectionType);
```